### ISSUE TITLE
When Bars or Stats are added, they are added on top of the list 

### ISSUE THREAD

**Issue by Phyndell** [uurshin/minotaure#65]
![image](https://github.com/uurshin/minotaure/assets/145913641/624d2959-c346-4d9f-88eb-970ff870f087)
A new Bar or a new stat is added to the top of the list regardless of alphabetical order

**Comment by uurshin** [comment]
Agreed. We should do the reverse : add the new bar or stat and the end (not that useful but it's better ergonomics)
